webapp/components/user_settings/user_settings_notifications.jsx

```diff
@@ -911,10 +936,7 @@ class NotificationsTab extends React.Component {
             );
         }
 
-        let pushNotificationSection;
-        if (global.window.mm_config.SendPushNotifications === 'true') {
-            pushNotificationSection = this.createPushNotificationSection();
-        }
+        let pushNotificationSection = this.createPushNotificationSection();
```

Comment:
Minior, but can you use `const` here instead of `let` if it's only being set once?